### PR TITLE
Wip dunfell

### PR DESCRIPTION
**Pull request by drewmoseley**
# External Contributor Checklist

🚨 Please review the [guidelines for contributing](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md) to this repository.

- [ ] Make sure that all commits have a [`Changelog`](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md#changelog-tags) tag. If nothing should be added to the Changelog, add a `Changelog: None` tag. If there is a change, add `Changelog: Commit`, or `Changelog: Title`, depending on what should be included in the changelog.

- [ ] Make sure that all commits are signed with [`git --signoff`](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work). Also note that the signoff author must match the author of the commit.

### Description

Various fixes from my recent projects.
These are not high priority but wanted to get them submitted on the branches I use for review.

### CODE REVIEW

File: meta-mender-qemu/recipes-kernel/linux/linux-yocto_%.bbappend [hunk 1]
```diff
@@ -1,14 +1,15 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
 # Use custom defconfig in order to enable use of the vexpress model.
-SRC_URI_append_vexpress-qemu = " file://defconfig \
+SRC_URI_append_vexpress-qemu = " file://vexpress-qemu-mender.cfg \
```
Comment (lluiscampos) [inline meta-mender-qemu/recipes-kernel/linux/linux-yocto_%.bbappend:4]: I don't know why, but this change makes one test when running `losetup --show -f /data/tmp-file-system`:
```
losetup: cannot find an unused loop device
```

Does it ring a bell? Have we accidentally changed something in the configuration?

Comment (drewmoseley) [reply meta-mender-qemu/recipes-kernel/linux/linux-yocto_%.bbappend:4]: Well CONFIG_BLK_DEV_LOOP is required: https://github.com/mendersoftware/meta-mender/commit/481bfc9abfcdac83447a1fc67a8342847b75363a

Not sure why diffconfig missed this. But I just reworked this patch to simply remove the bits that were generating warnings.  We still use a defconfig which would Ideally be exchanged for a config fragment but I had no luck making that happend.

File: meta-mender-commercial/recipes-mender/mender-client/mender-client_%.bbappend [hunk 2]
```diff
@@ -0,0 +1 @@
+# Deliberately empty bbappend file to remove bitbake warnings about no bb files
```
Comment (lluiscampos) [inline meta-mender-commercial/recipes-mender/mender-client/mender-client_%.bbappend:1]: As pointed out in the master PR (#1060) I don't think we need this file at all. Please remove or prove me wrong :)

Comment (lluiscampos) [reply meta-mender-commercial/recipes-mender/mender-client/mender-client_%.bbappend:1]: And follow-up accordingly in zeus (#1057) and warrior (#1058).

File: meta-mender-commercial/conf/layer.conf [hunk 3]
```diff
@@ -5,8 +5,8 @@
 BBPATH .= ":${LAYERDIR}"
 
 # We have recipes-* directories, add to BBFILES
-BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
-	${LAYERDIR}/recipes-*/*/*.bbappend"
+# Currently only bb files, no bbappends.
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb"
```
Comment (kacf) [inline meta-mender-commercial/conf/layer.conf:9]: I hope I remember this when the time comes to add one! :-)

**Pull request merged**